imap_processing/ena_maps/utils/corrections.py

```diff
@@ -344,10 +345,26 @@ def add_spacecraft_velocity_to_pset(
     - "sc_velocity": Spacecraft velocity vector (km/s) with dims ["x_y_z"]
     - "sc_direction_vector": Spacecraft velocity unit vector with dims ["x_y_z"]
     """
-    # Compute ephemeris time (J2000 seconds) of PSET midpoint
-    # epoch contains Pointing start time, and epoch_delta indicates the total
-    # duration of the Pointing
-    et = ttj2000ns_to_et(pset["epoch"].values[0] + pset["epoch_delta"].values[0] / 2)
+    # Hi and Lo need to use different methods for computing the Pointing
+    # midpoint time.
```

Comment:
Is this something we can add to the Lo pointing set to make it align with Hi? It seems like these should be identical between the instruments to me.